libr/bin/format/elf/elf.c

```diff
@@ -728,38 +725,32 @@ static RBinElfSection* get_section_by_name(ELFOBJ *eo, const char *section_name)
 	if (eo->sections_loaded) {
 		RBinElfSection *section;
 		r_vector_foreach (&eo->g_sections, section) {
-			if (!strncmp (section->name, section_name, ELF_STRING_LENGTH - 1)) {
+			if (!strcmp (section->name, section_name)) {
 				return section;
 			}
 		}
 	}
-
 	return NULL;
 }
 
-static const char *get_ver_flags(char *buff, size_t buff_size, ut32 flags) {
-	if (!flags) {
-		return "none";
+static char *get_ver_flags(ut32 flags) {
+	if (flags == 0) {
+		return strdup ("none");
 	}
-
-	buff[0] = 0;
-
+	RStrBuf *sb = r_strbuf_new ("");
```

Comment:
```suggestion
	RStrBuf *sb = r_strbuf_new (flags & VER_FLG_BASE ? "BASE" : "");
```

+ remove if (saves 1 allocation I think)